qutebrowser/config/configdata.yml

```diff
@@ -345,6 +345,46 @@ qt.chromium.experimental_web_platform_features:
     Chromium. By default, this is enabled with Qt 5 to maximize compatibility
     despite an aging Chromium base.
 
+qt.chromium.use_recommended_page_lifecycle_state:
+  type: Bool
+  default: false
+  backend: QtWebEngine
+  # yamllint disable rule:line-length
+  desc: >-
+    Use recommended page lifecycle state.
+
+    This puts webpages into one of three lifecycle states: active, frozen, or
+    discarded. Using the recommended lifecycle state lets the browser use less
+    resources by freezing or discarding web views when it's safe to do so.
+
+    This results in significant battery life savings.
+
+    Ongoing page activity is taken into account when determining the
+    recommended lifecycle state, as to not disrupt your browsing.
+
+    See the Qt documentation for more details: https://doc.qt.io/qt-6/qtwebengine-features.html#page-lifecycle-api
+  # yamllint enable rule:line-length
+
+qt.chromium.lifecycle_state_freeze_delay:
+  type: Int
+  default: 0
+  backend: QtWebEngine
+  desc: >-
+    The amount of time (in milliseconds) to wait before transitioning a page to
+    the frozen lifecycle state.
+
```

Comment:
We should explain a bit more about what that means for people who aren't familiar with the concept (like me before I looked at this PR!).

From the Qt docs:

> The frozen state is a low CPU usage state. In this state, most HTML task sources are suspended (frozen) and, as a result, most DOM event processing and JavaScript execution will also be suspended.